exist-core/src/main/java/org/exist/xmldb/LocalBinaryResource.java

```diff
@@ -121,26 +121,35 @@ Object getContent(final DBBroker broker, final Txn transaction) throws XMLDBExce
 
     private Object getContent(final Object res) throws XMLDBException {
         if(res != null) {
```

Comment:
I do prefer to handle the null case at the beginning 
```java
if (res == null) { return; }

Reply:
Can't this be even rewritten to a case?
```java
return switch (res) { case null,default -> return; … }

Reply:
can be in switch indeed; had to study it myself